### PR TITLE
Respond to the channel with "hello" immediately.

### PR DESCRIPTION
**Pull request by blister**
Can we respond to the DM channel? Are we even getting to that code?

### CODE REVIEW

File: dmHelper/dmHelper.py [hunk 1]
```diff
@@ -122,7 +122,7 @@ async def on_member_join(self, member: discord.Member):
     async def _message_listener(self, message: discord.Message):
         if not isinstance(message.channel, discord.DMChannel):
             return
```
Comment (BenjaminHunt) [inline dmHelper/dmHelper.py:124]: This should be tabbed in. Did we miss this before?

**Review by BenjaminHunt — CHANGES_REQUESTED** [review]
One change

File: dmHelper/dmHelper.py (hunk 1 above)
Comment (blister) [reply dmHelper/dmHelper.py:124]: I'm not seeing what you're talking about.

Comment (BenjaminHunt) [reply dmHelper/dmHelper.py:124]: It might be showing weird on mobile. Is the return indented?

Comment (BenjaminHunt) [reply dmHelper/dmHelper.py:124]: Yeah I think it's just displaying weird

Comment (blister) [reply dmHelper/dmHelper.py:124]: Yes

**Review by BenjaminHunt — APPROVED** [review]
Lgtm

**Pull request merged**